gensim/models/wrappers/fasttext.py

```diff
@@ -298,8 +299,25 @@ def load_dict(self, file_handle, encoding='utf8'):
                 char_byte = file_handle.read(1)
             word = word_bytes.decode(encoding)
             count, _ = self.struct_unpack(file_handle, '@qb')
-            assert self.wv.vocab[word].index == i, 'mismatch between gensim word index and fastText word index'
-            self.wv.vocab[word].count = count
+
+            if i == nwords and i < vocab_size:
+                """
+                To handle the error in pretrained vector wiki.fr (French).
```

Comment:
This format - 
```
"""
Some comments
"""
```
 - is generally reserved for docstrings. Regular comments would be preferable.